server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -1794,45 +1698,101 @@ private File buildWin32Resources(Map<String, Object> mergedAppContext) throws Ex
     }
 
     private void loadManifests() throws IOException, ExtenderException {
+        // Creates the contexts for each stage: extension and app
+        //  extension context: merge(platform, variant, extension, app)
+        //  app context:       merge(platform, variant, extensions, app)
```

Comment:
This function now does a lot of the heavy lifting, in preparing the individual contexts for extensions and the app.

Here is listed the order of operations in order to create the final context:
* extension context: `merge(platform, variant, extension, app)`
* app context:           `merge(platform, variant, extensions, app)`